hackclub.com.yaml

```diff
@@ -614,6 +614,10 @@ mcneil:
 - ttl: 1
   type: A
   value: 70.123.47.120
+mea:
```

Comment:
Hey! Vercel doesn't allow for this style of CNAME. Furthermore, unfortunately due to platform restrictions we can't add non-HQ sites on Vercel to the DNS. I recommend using Netlify as an alternative. Sorry :)

Reply:
Deployed to Netlify, please let me know if this works :p